app.js

```diff
@@ -140,7 +140,9 @@ if (redisUrl) {
 const SIX_HOURS = 1000 * 60 * 60 * 6;
 app.use(session({
 	cookie: {
-		// TODO ...cookieDefaults,
+		httpOnly: true,
```

Comment:
```suggestion
		httpOnly: Configuration.get('COOKIE__ HTTP_ONLY),
```

Reply:
no, this should not even be configurable, having it set to false is broken from the beginning